SelfUpdateExecutor/SelfUpdateExecutor.cs

```diff
@@ -134,7 +134,10 @@ static void execute(string sourcePath, string targetPath, int processId)
             }
 
             // Startup new launcher; failure is irresolvable
-            Process.Start(targetPath + LauncherExeFilename, "--patch-result=" + status);
+            List<string> files = Directory.GetFiles(targetPath).ToList();
+            Regex reg = new Regex(@"\\([\w\s-_]*?\.exe)");
```

Comment:
I'm not so sure this one's such a good idea; when I was playing around with the self updater earlier, I think I noticed it might not be cleaning up files if the current directory the launcher is in, is open. The main implication of this was that I ended up with binaries from both versions of the launcher somehow.

Reply:
then ur delete directory method is just not working and might need a workaround to instead delete the files in the directory